extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -327,9 +327,9 @@ export async function getDefaultPublishDeploymentOptions(project: ISqlProject):
 	const result = await schemaCompareService.schemaCompareGetDefaultOptions();
 	// re-include database-scoped credentials
 	if (getAzdataApi()) {
-		result.defaultDeploymentOptions.excludeObjectTypes.value = (result.defaultDeploymentOptions as mssql.DeploymentOptions).excludeObjectTypes.value?.filter(x => x !== mssql.SchemaObjectType.DatabaseScopedCredentials);
+		result.defaultDeploymentOptions.excludeObjectTypes.value = (result.defaultDeploymentOptions as mssql.DeploymentOptions).excludeObjectTypes.value?.filter(x => x !== constants.DatabaseScopedCredentials);
```

Comment:
is this filtering still needed since the publish dialog sets excludeObjectTypes to an empty array?

Reply:
it was added in https://github.com/microsoft/azuredatastudio/pull/12840 because there was different behavior between deploying a dacpac with the dacpac wizard and publishing the same project because of the different excluded object types

Reply:
Looks like till the date, the SC and the publish uses same deployOptions and needed the DatabaseScopedCredentials  exclusion... But now they both have diff defaults, SC(with 22 default exclude objects) and Publish(none), we can remove this line of code now!

Also found this Item created for that.... #12847

Reply:
As discussed, I have removed the logic for re-including the databaseScopedCredentials form the object types as the default objecttypes are empty and doesn't require the change anymore. Thanks